bagit.xml

```diff
@@ -1,819 +1,800 @@
+
+          <t>
+            For reasons of efficiency, a bag &may; be sent with a list of files to be
+            fetched and added to the payload before it can meaningfully be checked
+            for completeness.  An &optional; tag file called the fetch file
+            contains such a list.
+          </t>
+
+          <t>
+            The fetch file &must; be named "fetch.txt". Every file listed in
+            the fetch file &must; be listed in every
+            payload manifest. A fetch file &must-not; list any tag files.
+          </t>
+          <t>
+            Each line of a fetch file &must; be of the form:
+          </t>
+          <figure>
+            <artwork>url length filename</artwork>
+            <postamble>
+              where <spanx style="emph">url</spanx> identifies the file to be
+              fetched and must be an absolute URI as defined in
+              <xref target="RFC3986"/>, <spanx style="emph">length</spanx> is
+              the number of octets in the file (or "-", to leave it unspecified),
+              and <spanx style="emph">filename</spanx> identifies the
+              corresponding payload file, relative to the base directory.
+            </postamble>
+          </figure>
+
+          <t>
+            The slash character ('/') &must; be used as a path separator in
+            <spanx style="emph">filename</spanx>. One or more linear whitespace
+            characters (spaces or tabs) &must; separate these
+            three values, and any such characters in the <spanx style="emph">url</spanx>
+            &must; be percent-encoded <xref target="RFC3986"/>.  There is no
+            limitation on the length of any of the fields in the fetch file.
+          </t>
+
+          <t>
+            The fetch file allows a bag to be transmitted with
+            "holes" in it, which can be practical for several reasons.  For example,
+            it obviates the need for the sender to stage a large serialized copy of
+            the content while the bag is transferred to the receiver.  Also, this
+            method allows a sender to construct a bag from components that are either
+            a subset of logically related components (e.g., the localized logical
+            object could be much larger than what is intended for export) or
+            assembled from logically distributed sources (e.g., the object components
+            for export are not stored locally under one filesystem tree).
+          </t>
+
+        </section>
+        <!-- /Fetch File -->
+        <section title="Other Tag Files" anchor="sec-other-tag-files">
+          <t>
+            A bag &may; contain other tag files that are not defined by this
+            specification.
+
+            Implementations &must; perform standard checksum validation on any tag file
+            which is listed in a tag manifest but &must; otherwise ignore their contents.
```

Comment:
Agree to remove the whole "but" clause. They can do with the remaining tag-files as they like, even during validation. 

For instance if the BagIt profile is https://w3id.org/ro/bagit/profile then a conformant implementation may also want to check that `manifest/metadata.json` is present and valid according to JSON-LD specification.   (If that fails, the bag is still "valid" according to BagIt spec, but not from the RO Bagit profile).

We could add a similar "opaque" clause as for the payload files, but then it gets tricky because SOME tagfiles have meaning for this spec.